### PR TITLE
Fix 2 Cannot Create Graphql APIs

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
Fix issue #7032

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 1]
```diff
@@ -952,4 +890,65 @@ private OpenAPI getOpenAPI(String oasDefinition) throws APIManagementException {
...
+
+        resource.setVerb(APIConstants.HTTP_GET);
+        Operation getOperation = createOperation(resource);
+        resource.setVerb(APIConstants.HTTP_POST);
+        Operation postOperation = createOperation(resource);
+
+        //get operation
+        Parameter getParameter = new Parameter();
+        getParameter.setName(APIConstants.GRAPHQL_QUERY);
+        getParameter.setIn("query");
```
Comment (Rajith90) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Lets try to use constant strings

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 2]
```diff
@@ -952,4 +890,65 @@ private OpenAPI getOpenAPI(String oasDefinition) throws APIManagementException {
...
+        getOperation.addParametersItem(getParameter);
+
+        //post operation
+        RequestBody requestBody = new RequestBody();
+        requestBody.setDescription("Query or mutation to be passed to graphQL API");
+        requestBody.setRequired(true);
+
+        JSONObject typeOfPayload = new JSONObject();
+        JSONObject payload = new JSONObject();
+        typeOfPayload.put(APIConstants.TYPE, "string");
```
Comment (Rajith90) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Lets use constant string

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 1 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed in https://github.com/wso2/carbon-apimgt/pull/8244

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java (hunk 2 above)
Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed in https://github.com/wso2/carbon-apimgt/pull/8244

**Pull request merged**